tests/findAllComponents.spec.ts

```diff
@@ -66,16 +66,17 @@ describe('findAllComponents', () => {
     const wrapper = mount(RootComponent)
 
     expect(wrapper.findAllComponents('.in-root')).toHaveLength(1)
-    expect(wrapper.findAllComponents('.in-root')[0].vm.$options.name).toEqual(
-      'NestedChild'
-    )
+    expect(
+      wrapper.findAllComponents<DefineComponent>('.in-root')[0].vm.$options.name
```

Comment:
I like that when you're doing something **nasty** (working outside `WrapperLike` interface which is unified across `VueWrapper` and `DOMWrapper` - you need to explicitly specify "hey, I know that I'm doing, there will be no functional component here". Type safety as it is!